QuickStarts/Finding Fraudulent Communities with Neo4j/Louvain-Community-Detection.md

```diff
@@ -195,10 +212,10 @@ CALL neo4j_graph_analytics.graph.louvain('CPU_X64_XS', {
 Our resulting table assigns a community id to each node based on their connections to other nodes in the graph.
 
 ```sql
-select community,   COUNT(*) AS community_size, 
-from p2p_demo.public.p2p_users_vw_lou
-group by community
-order by community_size desc;
+SELECT community, COUNT(*) AS community_size, 
```

Comment:
The table above is ill-formed, |=== and ===| are missed

Reply:
No idea on how to fix this.